public/authenticate/auth-guides/pass-params-idp.md

```
@@ -0,0 +1,140 @@
+    ></your_kinde_client_id
+  ></your_kinde_subdomain
+>
+```
+
+In this case both Kinde and the IDP use the parameter name `login_hint` so the configuration is the same on both sides:
+Add this to the connection configuration:
+
+}
+}
+
+````
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)